src/automations/index.js

```diff
@@ -71,9 +71,20 @@ export default class Automations {
       throw new Error('The "data" argument must be an object.')
     }
 
+    if (executionId !== undefined && typeof executionId !== 'string') {
+      throw new Error('The "executionId" argument must be a string.')
+    }
+
+    const query = {}
+
+    if (executionId !== undefined) {
```

Comment:
If an empty string is not a valid `executionId`, then we can (and should) check just for the `executionId` presence here:

```js
if (executionId) {
  query.executionId = executionId
}
```

Reply:
\+